app-misc/geoclue/geoclue-2.4.7.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+GCONF_DEBUG="no"
+
+inherit gnome2 systemd user versionator
+
+MY_PV=$(get_version_component_range 1-2)
+DESCRIPTION="A geoinformation D-Bus service"
+HOMEPAGE="https://freedesktop.org/wiki/Software/GeoClue"
+SRC_URI="https://www.freedesktop.org/software/${PN}/releases/${MY_PV}/${P}.tar.xz"
+
+LICENSE="LGPL-2"
+SLOT="2.0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd"
+IUSE="+introspection +modemmanager zeroconf"
+
+RDEPEND="
+	>=dev-libs/glib-2.34:2
+	>=dev-libs/json-glib-0.14
+	>=net-libs/libsoup-2.42:2.4
+	sys-apps/dbus
+	introspection? ( >=dev-libs/gobject-introspection-0.9.6:= )
+	modemmanager? ( >=net-misc/modemmanager-1 )
+	zeroconf? ( >=net-dns/avahi-0.6.10 )
+	!<sci-geosciences/geocode-glib-3.10.0
+"
+DEPEND="${RDEPEND}
+	dev-util/gdbus-codegen
+	>=dev-util/gtk-doc-am-1
+	>=dev-util/intltool-0.40
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-2.4.1-fix-GLIBC-features.patch
```

Comment:
If this ebuild goes EAPI version 6, patches must be defined in the `PATCHES` array. Also `default` must be called in the `src_prepare()` phase.